CHANGELOG.md

```diff
@@ -5,6 +5,106 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 8.7.0
+
+_Jul 3, 2025_
+
+We'd like to extend a big thank you to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 📊 Add `useChartProApiRef` for easier access to the API
```

Comment:
Kind of worth a highlight since it's an API change.
```suggestion
- 📊 Add `useChartProApiRef` for easier access to the API
- 📆 Support different start and end `referenceDate` props on range components
```